project/AkkaDependency.scala

```diff
@@ -54,15 +55,28 @@ object AkkaDependency {
               else moduleRef % config
 
             project.dependsOn(withConfig)
-          case Artifact(akkaVersion) =>
-            project.settings(libraryDependencies += {
-              if (config == "")
-                "com.typesafe.akka" %% module % akkaVersion
-              else
-                "com.typesafe.akka" %% module % akkaVersion % config
-            })
+          case Artifact(akkaVersion, akkaSnapshot) =>
+            project.settings(
+              libraryDependencies += {
+                if (config == "")
+                  "com.typesafe.akka" %% module % akkaVersion
+                else
+                  "com.typesafe.akka" %% module % akkaVersion % config
+              },
+              resolvers ++= (if (akkaSnapshot) Seq("Akka Snapshots" at "https://repo.akka.io/snapshots") else Nil)
+            )
         }
       }
       else project // return unchanged
   }
+
+  private def determineLatestSnapshot(prefix: String = ""): String = {
+    import sbt.librarymanagement.Http.http
+    import gigahorse.GigahorseSupport.url
+    import scala.concurrent.Await
+    import scala.concurrent.duration._
+
+    val body = Await.result(http.run(url("https://repo.akka.io/snapshots/com/typesafe/akka/akka-actor_2.12/")), 10.seconds).bodyAsString
+    """href="([^?/].*?)/"""".r.findAllMatchIn(body).map(_.group(1)).filter(_.startsWith(prefix)).toList.last
```

Comment:
Seems a bit brittle but might be good enough. How sure are we that the listing is ordered by version? I.e. the dir listing is ordered lexically and lexical order == version order.

I think we should at least log the result here or somewhere else so it's easier to figure out if somethings goes wrong.

Reply:
It's ordered by file timestamp right now. That's obviously brittle, though most approaches are... I'd propose using this until we find out we need something better ;).

Reply:
Ok, timestamp sounds good, maybe better than lexical.